platforms/web/scripts/build-tools/buildTools.ts

```diff
@@ -90,6 +86,33 @@ export const generateIconTags = (basePath: string, favIconSizes: number[], apple
   return [...favIconTags, ...appleIconTags].join('\n');
 };
 
+export const getRelatedApplications = ({
+  appleAppId,
+  googleAppId,
+}: {
+  appleAppId?: string | undefined;
+  googleAppId?: string | undefined;
+} = {}): ManifestOptions['related_applications'] => {
+  const relatedApplications = [];
+
+  if (appleAppId) {
+    relatedApplications.push({
+      platform: 'itunes',
+      url: `https://apps.apple.com/nl/app/${appleAppId}`,
```

Comment:
Can we avoid using `nl` in the url?

Reply:
Apologees, the locale was not supposed to get it in there. 😅